lib/absinthe/lexer.ex

```diff
@@ -8,6 +8,8 @@ defmodule Absinthe.Lexer do
   @space 0x0020
   @unicode_bom 0xFEFF
 
+  @stopped_at_token_limit ":stopped_at_token_limit"
+
```

Comment:
Because `nimble_parsec` specific errors as `String.t()`.

Another solution is to pull request there so the error reason can be `term()`, for example.